scala_proto/scala_proto.bzl

```diff
@@ -81,8 +81,8 @@ def scala_proto_repositories():
 
     native.maven_jar(
         name = "scala_proto_rules_scalapb_lenses",
-        artifact = "com.trueaccord.lenses:lenses_2.11:0.4.12",
-        sha1 = "c5fbf5b872ce99d9a16d3392ccc0d15a0e43d823",
+        artifact = "com.trueaccord.lenses:lenses_2.12:0.4.12",
```

Comment:
if we used `scala_mvn_artifact` here we wouldn't have this change and others in the file.